sdk/src/test/java/com/finbourne/lusid/tutorials/marketdata/Instruments.java

```diff
@@ -206,22 +206,23 @@ public void list_instruments_by_Identifier_type() throws ApiException {
 
     @Test
     public void edit_instrument_property() throws ApiException {
-
+        String figi = "BBG000C6K6G9";
         //  Create the property value
         PropertyValue   propertyValue = new PropertyValue().labelValue("Telecoms");
         String propertyKey = String.format("Instrument/%s/CustomSector", TutorialScope);
 
         //  Get the LusidInstrumentId (LUID)
```

Comment:
Should remove this getting of the instrument for the LUID altogether as you did with the CSharp SDKs